File: API_MediBridge/API_MediBridge/Models/PatientDoctor.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Web;

namespace API_MediBridge.Models
{
    public class PatientDoctor
    {
        


        [Key]
        public int PatientDoctorId {  get; set; }

        

        public int DoctorsId {  get; set; }

        [ForeignKey("User")]
        public int UsersId {  get; set; }

        public virtual Doctors Doctors { get; set; }
        public virtual Users User { get; set; }

    }
}

File: MediBridge/src/app/classes/patient-doctor.ts
import { Doctor } from './doctor';
import { Patient } from './patient';
export class PatientDoctor {
  constructor(
    public DoctorsId: number,
    public UsersId: string,
    public PatientDoctorId?: number,
    public Doctor?: Doctor,
    public Patient?: Patient
  ) {}
}
